### DISCUSSION TITLE
transaction in simblock

### DISCUSSION THREAD

**Discussion by s77a85m** [community/community#126206] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello. I am using SIMBlock simulator, but I need to know the transactions, the transaction repository, how the nodes select transactions and the transactions related matters, are the transactions embedded in this simulator at all? If the answer is negative, can it be added?

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
